src/assets/scripts/install-wsl-helpers

```diff
@@ -12,3 +12,4 @@ set -o errexit -o nounset
 mkdir -p "/mnt/wsl/rancher-desktop/bin/"
 cp "${1}" "/mnt/wsl/rancher-desktop/bin/nerdctl"
 chmod u+s "/mnt/wsl/rancher-desktop/bin/nerdctl"
+ln -fs "/mnt/wsl/rancher-desktop/bin/nerdctl" "/mnt/wsl/rancher-desktop/bin/docker"
```

Comment:
`/mnt/wsl/rancher-desktop/bin/docker` is never used, but that's harmless.

Reply:
Pulling it out then